lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'code',
+    schema: [{
+      type: 'object',
+      properties: {
+        'firstline': {
+          type: 'number',
+          minimum: 0
+        },
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'number',
+          minimum: 1
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0]
+    const multilineMaximum = configuration.multiline
+    const singlelinemMaximum = configuration.singleline
+    const canHaveFirstLine = configuration.firstline !== 0
```

Comment:
I'd go with `C` but still keep it as number, e.g.:
```
{
    "singleline": 3,
    "multiline": {"max": 1, "maxInFirstLine": 2}
}
```
So it's possible to have two properties in the first line but the rest in new lines in capacity of 1 per line, like here:
```
<input name="someField" class="input"
  v-model="message"
  placeholder="Lorem ipsum"
>
```
What do you think @filipalacerda @mysticatea ?